network/vpc/reference-content/understanding-routing.mdx

```diff
@@ -0,0 +1,90 @@
+---
+meta:
+  title: Understanding VPC routing
+  description: This page provides detailed information about the VPC routing feature, which connects Private Networks within the same VPC.
+content:
+  h1: Understanding VPC routing
+  paragraph: This page provides detailed information about the VPC routing feature, which connects Private Networks within the same VPC.
+tags: vpc routing route-table private-network managed-route automatic-route local-subnet-route default-route
+dates:
+  validation: 2024-05-28
+  posted: 2024-05-28
+categories:
+  - network
+---
+
+## Overview
+
+VPC routing allows Private Networks in the same VPC to communicate with each other via automatically-created and managed routes.
+
+Every time you create a Private Network in a VPC with routing activated, a managed route is automatically created and added to the VPC’s route table. This route allows the VPC to automatically route packets to resources attached to that Private Network, even if they originate from a resource attached to a different Private Network on the VPC.
+
+Managed routes are also automatically added to the VPC’s routing table when you attach a Public Gateway to a Private Network, and tell it to advertise the default route. This type of managed route allows traffic on the given Private Network to be forwarded to addresses outside the VPC (i.e. the public internet) via the Public Gateway.
```

Comment:
```suggestion
Managed routes are also automatically added to the VPC’s routing table when you attach a Public Gateway to a Private Network, and tell it to advertize the default route. This type of managed route allows traffic on the given Private Network to be forwarded to addresses outside the VPC (i.e. the public internet) via the Public Gateway.
```

Reply:
British Roro spotted hehe

Reply:
Mais non ! Pour une fois ils gardent le s
https://english.stackexchange.com/questions/341707/should-advertised-be-spelled-with-a-z-in-american-english